source/frontend/src/features/mapSideBar/acquisition/tabs/fileDetails/detail/AcquisitionSummaryView.tsx

```diff
@@ -137,6 +142,22 @@ const AcquisitionSummaryView: React.FC<IAcquisitionSummaryViewProps> = ({
             </StyledLink>
           </SectionField>
         )}
+        {ownerSolicitor?.organization && (
```

Comment:
Looks like you are missing a requirement. If there is only one primary contact, it should be automatically selected. as per: 

2. Screenshot 2: User cannot select different primary contact, if there is only ONE attached to linked organization